rtengine/camconst.cc

```diff
@@ -637,18 +643,17 @@ CameraConstantsStore::parse_camera_constants_file(Glib::ustring filename_)
 
             Glib::ustring make_model(ji->valuestring);
             make_model = make_model.uppercase();
-            std::map<Glib::ustring, CameraConst *>::iterator existingccIter = mCameraConstants.find(make_model);
 
-            if (existingccIter == mCameraConstants.end()) {
-                // add the new CamConst to the map
-                mCameraConstants.insert(std::pair<Glib::ustring, CameraConst *>(make_model, cc));
+            std::pair<std::map<Glib::ustring, CameraConst *>::iterator, bool> ret;
+            ret = mCameraConstants.insert(std::pair<Glib::ustring, CameraConst *>(make_model, cc));
```

Comment:
@heckflosse May I suggest one small change here? This is a place where `auto` really shines. So you could convert it to:
```c++
const auto ret = mCameraConstants.emplace(make_model, cc);
```
Best,
Flössie